src/ert/run_models/multiple_data_assimilation.py

```diff
@@ -199,7 +199,7 @@ def _simulateAndPostProcess(
     ) -> int:
         iteration = run_context.iteration
 
-        phase_string = f"Running simulation for iteration: {iteration}"
+        phase_string = f"Running experiment for iteration: {iteration}"
```

Comment:
Here it actually makes sense to say simulation and not experiment, since this describes only the part of the experiment where we run the simulations. 
```suggestion
        phase_string = f"Running simulation for iteration: {iteration}"
```